cibyl/utils/files.py

```diff
@@ -52,6 +59,6 @@ def get_first_available_file(filenames, file_check=_is_file_available):
             return filename
 
         LOG.debug('%s: %s', 'file not found', filename)
-
+        # raise NoFileFound()
```

Comment:
Please remove this comment, it leads to a mistake. If we were to use the 'NoFileFound' exception on this class, then we would use it to replace the 'return None' at the end.